.github/workflows/ci.yml

```diff
@@ -129,3 +144,87 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: ${{ steps.registry_refs.outputs.tags }},mode=max
           cache-to: ${{ steps.registry_refs.outputs.tags }},mode=max
+
+  integration-tests:
+    needs: [determine-changes, lint, test, build-container]
+    runs-on: ubuntu-latest
+    if: ${{ needs.determine-changes.outputs.non-test-changes }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Extract Branch Name
+      run: |
+        if [ "$GITHUB_HEAD_REF" != "" ]; then
+          echo "BRANCH_NAME=$GITHUB_HEAD_REF" >> $GITHUB_ENV
+        else
+          echo "BRANCH_NAME=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+        fi
+
+    - name: Pull Public Docker Image from GHCR
+      run: |
+        docker pull ghcr.io/sage-bionetworks/genie:${{ env.BRANCH_NAME }}
+
+    - name: Start Docker Container
+      run: |
+        docker run -d --name genie-container \
+          -e SYNAPSE_AUTH_TOKEN="${{ secrets.SYNAPSE_AUTH_TOKEN }}" \
+          ghcr.io/sage-bionetworks/genie:${{ env.BRANCH_NAME }} \
+          sh -c "while true; do sleep 1; done"
```

Comment:
whats the motivation of this `while true` statement?

Reply:
Hmm from my understanding I thought that we had to keep the container "open" while running the code to keep it all in one container run/same state persistence. However thinking about it now, it should probably mirror nextflow where each step is probably separate docker run. 

I also wonder if this is the reason why I'm seeing differences in the outputs compared to outputs when I run the nextflow pipeline (something I am investigating ...)

Reply:
Differences in outputs?!  What are the differences?

Reply:
All in the clinical consortium files. It's dropping records for some reason.

Reply:
It was due to the different seq_dates used :')